addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2869,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
```

Comment:
Nope, see example 5 on the link you gave. Its a setting name therefore a title of sorts.